src/extension.ts

```diff
@@ -21,6 +21,17 @@ import { ENV0_ENVIRONMENTS_VIEW_ID } from "./common";
 
 let logPoller: NodeJS.Timeout;
 let environmentPollingInstance: NodeJS.Timer;
+let _context: vscode.ExtensionContext;
+export let environmentsTree: vscode.TreeView<Environment>;
+export let environmentsDataProvider: Env0EnvironmentsProvider;
+// this function used by tests in order to reset the extension state after each test
+export const _reset = async () => {
```

Comment:
this code is here because i want to reset the extension after each test.
Since I couldn't find an vscode api to deactivate/disable the extension i created this function.
i found only a feature request for programmatically deactivate an extension [here](https://github.com/microsoft/vscode/issues/147641).